src/tests/ov_helpers/ov_snippets_models/src/subgraph_mha.cpp

```diff
@@ -204,7 +204,7 @@ std::shared_ptr<ov::Model> MHASplitMFunction::initReference() const {
 
         if (ov::shape_size(shape) > 1) {
             ov::Shape reshape_shape = shape;
-            reshape_shape.insert(reshape_shape.cbegin() + rank - 3, 1);
```

Comment:
Thank you for the fix!

Could you also please update the lines [L85](https://github.com/openvinotoolkit/openvino/blob/9403be5237a80e1a5e618a6792d00fd71ca4fb7b/src/tests/ov_helpers/ov_snippets_models/src/subgraph_mha.cpp#L85) and [L291](https://github.com/openvinotoolkit/openvino/blob/9403be5237a80e1a5e618a6792d00fd71ca4fb7b/src/tests/ov_helpers/ov_snippets_models/src/subgraph_mha.cpp#L291)?

Reply:
Applied.